tools/darwin/packaging/osx/mkdmg-osx.sh.in

```diff
@@ -34,7 +34,7 @@ fi
 ARCHIVE=${PACKAGE}_${VERSION}-${REVISION}_macosx-intel-${ARCHITECTURE}
 
 echo Creating $PACKAGE package version $VERSION revision $REVISION
-${SUDO} rm -rf $DIRNAME/$ARCHIVE
+rm -rf $DIRNAME/$ARCHIVE
```

Comment:
@Memphiz: Surprisingly this failed on jenkins with `sudo: no tty present and no askpass program specified`. Do you have any idea what's causing this and how to fix it? Removing sudo causes issues when rebuilding the image locally. It seems to work on the non-cmake build though.